recipes/lixtools/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "lixtools" %}
+{% set version = "2021.6.9.0" %}
+{% set sha256 = "d12e77bfc770ddb9137345c054a6c02afba29bac72768212615126b94b062c4b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib
+    - numpy
+    - openpyxl >=3
+    - pandas
+    - pillow
+    - py4xs
+    - python-barcode
+    - qrcode
+
+test:
+  imports:
+    - lixtools
+  commands:  # [not win]
+    - python -c 'import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename("lixtools", "plate_label_template.html"))'  # [not win]
+    - python -c 'import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename("lixtools", "template_report.ipynb"))'  # [not win]
```

Comment:
I am against excluding tests for Windows. Could you try to fix it instead? On Windows CLI, single quotes are not accepted. Please use double-quotes. An easy way to test it on your Windows machine is to install the package locally and try this `python -c ...` string.